snapcraft/internal/pluginhandler/_part_environment.py

```diff
@@ -88,6 +88,7 @@ def get_snapcraft_part_directory_environment(
             {
                 "SNAPCRAFT_PART_BUILD": part.part_build_dir,
                 "SNAPCRAFT_PART_BUILD_WORK": part.part_build_work_dir,
+                "SNAPCRAFT_PART_BUILD_PLUGIN": part.part_build_plugin_dir,
```

Comment:
I'm iffy on this name, but I struggle to think of something better that still matches the established patterns. For example, I like "plugindir" better here, but that doesn't match "work". @sergiusens and @cjp256, got any better ideas?